plugins/calendar/calendar.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"log"
+	"strings"
+
+	"github.com/disiqueira/ultraslackbot/pkg/bot"
+	"github.com/disiqueira/ultraslackbot/pkg/slack"
+	"github.com/disiqueira/ultraslackbot/plugins/calendar/cmd"
+	"github.com/disiqueira/ultraslackbot/plugins/calendar/pkg"
+)
+
+const (
+	name = "Calendar"
+)
+
+type (
+	calendarPlugin struct{}
+)
+
+var myCalendar = pkg.CalendarImp{}
+var argParser = cmd.ArgParser{Calendar: &myCalendar}
```

Comment:
Usually, it's not a good idea to declare package level variables. Would be better to declare the parser as an attribute of the calendarPlugin struct, also would be better to use interfaces here instead of the struct types directly. It can be re-write as something like this:

```
type (
	calendarPlugin struct{
		argParser Parser
	}
	Parser interface {
		...
	}
)```

and you can use the Start() func to initialize the argParser and pass the Calendar as a parameter to the constructor:

```
func (c *calendarPlugin) Start() error {
        c.argParser = parser.New(calendar.New())
}
```